### PR TITLE
Fix build issues with 0.4.0

### LINKED ISSUE #565: Build is broken on windows
Getting lots of libssh issues during compile, seems tied to the commit where libssh was updated from 1.4.3 to 1.5.0. Trying a rollback now.

### PR DESCRIPTION
**Pull request by maxkorp**
Fixes #565 
Waiting to merge until we can get the false positives resolved

### CODE REVIEW

**Comment by johnhaley81** [comment]
@maxkorp I think the false positives in CI should be a different PR. We should get this merged in ASAP IMO.

File: package.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "lcov-result-merger": "^1.0.2",
     "lodash": "^3.8.0",
     "mocha": "^2.2.4",
-    "nan": "~1.7.0",
+    "nan": "~1.8.0",
```
Comment (tbranyen) [inline package.json]: Isn't latest NaN, 1.8.4? If that works, can we change this to `^1.8.4`?

Comment (johnhaley81) [inline package.json]: Yeah, np. It was downloading 1.8.4 but we should up that regardless.

**Comment by johnhaley81** [comment]
Seg faults in iojs on linux :(

**Comment by maxkorp** [comment]
I'd like to get it working on iojs in linux before merging, but I'm cool waiting on the false positives thing. I can take a crack at it in a few.

**Comment by johnhaley81** [comment]
I'm unable to repo the seg fault on linux. @tbranyen could you give this a try?

**Comment by maxkorp** [comment]
So apparently its an issue with iojs 2+. Any ideas/thoughts?

**Comment by johnhaley81** [comment]
Only in linux, OSX looks to be good. Have you tried windows?

**Comment by johnhaley81** [comment]
Removed the bumping nan and supporting io.js from this PR.

**Pull request merged**